### DISCUSSION TITLE
Is there any method setting the default boot option?

### DISCUSSION THREAD

**Discussion by lovelock** [sileshn/Ryzentosh#38] [Q&A] [answered]
I don't have dual boot disks, every time it reboots it'll stuck at BootPicker maybe? There are 3 options, the big one in the middle is the os I am going to boot into, and 2 big buttons behind it which are restart and shutdown. I searched the internet everyone says press Ctrl-Enter will select the OS as the default one and next time it won't get stuck here, but it never works.

When Ctrl is pressed, there is a red labeled "SET" shown behind the OS option, and press Enter it'll boot into it. Next time it reboot, it still stop at that stage and let me choose.

**Comment by sileshn** [comment, marked as answer]
Set ShowPicker to false

**Reply by lovelock** [reply]
Problem solved. Thanks!

**Comment by lovelock** [comment]
God you're  genius！

Besides there's another question, with your great EFI files, my desktop is upgraded to macOS Sonoma 14.4.1, what's wired is that the NIC (RT8111 or some other model) does not work anymore. There are Wi-Fi and Bluetooth items recoginzed in the tab of PCIE in the system report, but Wi-Fi and airdrop aren't able to be turn on( the button in the menu bar is greyed out), do you have any clues?

**Comment by sileshn** [comment]
Send me the details of the wifi/bluetooth card you use. If its built-in to your motherboard, then chances are it will not work

**Reply by lovelock** [reply]
No it's not built-in, it's Fenv T919(BCM94360CD) plugged in a PCIEx1 slot. It has been working quite well since I used your EFI files about 3 years ago, it broke after upgraded to macOS 14.4.1. I did not change anything from your EFI except the Serial number.
![image](https://github.com/sileshn/Ryzentosh/assets/2208731/0ed7d814-1584-460e-ac64-a67157c71cda)

**Comment by sileshn** [comment]
Ya, Apple removed support for Broadcom chipset in Sonoma. Only intel based ones are working now. Your best bet is to get a intel chip based card. You can also try the steps from the link below and see if it works for you.

https://www.reddit.com/r/hackintosh/comments/170q5wu/enable_wifi_in_sonoma_with_fenvi_t919

**Reply by lovelock** [reply]
Oh god now I know the reason why you changed your Wi-Fi adapter. Several years ago only Broadcom chipset was supported and now things got completely reversed. 
<img width="737" alt="image" src="https://github.com/sileshn/Ryzentosh/assets/2208731/0e974c91-c892-49a0-90ba-b0382e05352e">
